js/script.js

```diff
@@ -0,0 +1,106 @@
+var Counter = React.createClass({
+
+        increment: function() {
+            this.setState({
+                counter: this.state.counter + 1
```

Comment:
zachęcam do poczytania https://medium.freecodecamp.org/functional-setstate-is-the-future-of-react-374f30401b6b - w wolnej chwili możesz przerobić swoje `setState`